### PR TITLE
adding life stage expression widget

### PR DESCRIPTION
**Pull request by adamjohnwright**
This has been tested with life stage: WBls:0000025

I will need curator feedback to make sure things are pulled from the right places. I think this should be done when it is on staging. 

Let me know if you have any code related fixes / cleanup ideas.

### CODE REVIEW

File: src/rest_api/classes/clone/widgets/overview.clj [hunk 1]
```diff
@@ -118,22 +119,7 @@
...
-                                     :class "expr_pattern"
-                                     :label (str "Expression pattern for "
-                                                 (if-let [n (or (:gene/public-name gene)
-                                                                (:gene/id gene))]
-                                                   n
-                                                   "" ))
-                                     :id (:expr-pattern/id ep)}
-                :reference (when-let [paper (first (:expr-pattern/reference ep))]
-                             (:paper/id (:expr-pattern.reference/paper paper)))
-                :gene (if (empty? gene) nil (pack-obj gene))
```
Comment (mgrbyte) [inline src/rest_api/classes/clone/widgets/overview.clj:134]: Maybe `(if-not (empty? gene) (pack-obj gene)`

File: src/rest_api/classes/expr_pattern/core.clj [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  (let [h (first (:expr-pattern/gene ep))
+	gene (:expr-pattern.gene/gene h)]
+    {:certainty (generic-functions/certainty h)
+     :expression_pattern {:taxonomy "all"
+			  :class "expr_pattern"
+			  :label (str "Expression pattern for "
+			  (if-let [n (or (:gene/public-name gene)
+					 (:gene/id gene))]
+			  n
+			  "" ))
```
Comment (mgrbyte) [inline src/rest_api/classes/expr_pattern/core.clj]: extraneous space after empty string literal

Comment (mgrbyte) [reply src/rest_api/classes/expr_pattern/core.clj]: indentation (if-let should line up under "Expression...)

File: src/rest_api/classes/expr_pattern/core.clj [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+			  :id (:expr-pattern/id ep)}
+     :reference (some->> (:expr-pattern/reference ep)
+			 (map :expr-pattern.reference/paper)
+			 (map :paper/id)
+			 (first))
+     :gene (if (empty? gene) nil (pack-obj gene))
```
Comment (mgrbyte) [inline src/rest_api/classes/expr_pattern/core.clj]: same if-not pattern as above

**Review by mgrbyte — CHANGES_REQUESTED** [review]
Minor suggestions added, otherwise :+1:

File: src/rest_api/classes/clone/widgets/overview.clj (hunk 1 above)
Comment (adamjohnwright) [reply src/rest_api/classes/clone/widgets/overview.clj:134]: I thought it looked odd too. It was old code that I was moving. I am fixing it

File: src/rest_api/classes/expr_pattern/core.clj (hunk 2 above)
Comment (adamjohnwright) [reply src/rest_api/classes/expr_pattern/core.clj]: I am not sure why vim lined it up that way. I just tried to get vim to realign it and it is correct now.

**Review by sibyl229 — APPROVED** [review]

**Review by mgrbyte — APPROVED** [review]

**Pull request merged**